.github/workflows/release-connect-v9-rollback.yml

```diff
@@ -0,0 +1,26 @@
+name: "[Release] connect v9 rollback"
+
+permissions:
+  id-token: write # for fetching the OIDC token
+  contents: read # for actions/checkout
+
+on:
+  workflow_dispatch:
+
+jobs:
+  rollback-connect-production:
+    if: startsWith(github.ref, 'refs/heads/release/connect/')
+    environment: production
+    name: "Create rollback copy of connect.trezor.io"
+    runs-on: ubuntu-latest
+    env:
+      LATEST_VERSION: 9
+    steps:
+      - name: Configure aws credentials
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          role-to-assume: arn:aws:iam::538326561891:role/gh_actions_trezor_suite_prod_deploy
+          aws-region: eu-central-1
+
+      - run: |
```

Comment:
i would also name this step.

Reply:
Name like here https://github.com/trezor/trezor-suite/pull/12192/commits/ba951cebf16a76d8dd6d23fded1bfc5ab571e733 ?

Reply:
yeah this should be fine :)